docs/configuration/install.md

```diff
@@ -192,6 +192,11 @@ Usually Apache ships with a default configuration (e.g. ```/etc/apache2/sites-en
         AllowOverride All
         Require all granted
     </Directory>
+    <Directory /var/drbd/www/html/ilias/setup/>
```

Comment:
I think we overlooked some kind of copy-paste error above, which continues here: Since the document root is `/var/www/html/ilias` in the example, the setup directory in question should be located at `/var/www/html/ilias/setup`. I don't know where the `drbd` part in the path above came from, but it is most probably wrong.

Reply:
fixed